app/src/main/res/values-fr/strings.xml

```diff
@@ -176,12 +176,11 @@
     <string name="tip_t">Conseils : Faites un poing avec votre pouce coincé entre votre index et votre majeur.</string>
     <string name="tip_u">Conseils : Tenez votre index et majeur vers le haut et proches l’un de l’autre, tandis que les autres doigts sont repliés vers le bas.</string>
     <string name="tip_v">Conseils : Tenez votre index et majeur vers le haut et séparés en forme de V, tandis que les autres doigts sont repliés vers le bas.</string>
-    <string name="tip_w">Conseils : Tenez votre index, majeur et annulaire vers le haut, les écartant, tandis que les autres doigts sont repliés vers le bas.</string>
+    <string name="tip_w">Conseils : Tenez votre index, majeur et annuFlaire vers le haut, les écartant, tandis que les autres doigts sont repliés vers le bas.</string>
```

Comment:
```suggestion
    <string name="tip_w">Conseils : Tenez votre index, majeur et annulaire vers le haut, les écartant, tandis que les autres doigts sont repliés vers le bas.</string>
```
I think the "F" was added by mistake.

Reply:
Yup, I've finally found out how to stop Android Studio from autosaving everything. I'll stop commiting random commands into files now.